client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -614,17 +629,20 @@ export default React.createClass( {
 				{ ! this.props.isCompact && fullNotice }
 			</Notice>
 		);
-	},
+	}
 
-	componentWillMount: function() {
+	componentWillMount() {
 		if ( ! this.props.domains && ! this.props.domain ) {
 			debug( 'You need provide either "domains" or "domain" property to this component.' );
 		}
-	},
+	}
 
-	render: function() {
+	render() {
 		debug( 'Domains:', this.getDomains() );
 		const notices = this.getPipe().map( renderer => renderer() ).filter( notice => notice );
-		return notices.length ? <div className="site__notices">{ notices }</div> : null;
+		return notices.length ? <div>{ notices }</div> : null;
```

Comment:
The `site__notices` style is not defined anywhere anymore (plus, it was causing a linter warning).